### ISSUE TITLE
can't configure on Fedora 34 Gnome 40

### ISSUE THREAD

**Issue by airtonix** [bitgapp/Cursr#14]
### Describe the Bug/Issue

Clicking "Open" doesn't result in any reaction.

### Steps to Reproduce

1. Go to https://github.com/bitgapp/Cursr/releases/tag/v1.4.1
2. download the appimage
3. mark the appimage as executable
4. run the appimage
5. see the cursr icon show up in the top icons (you need to have installed some kind of top icons gnome shell extension)
6. Click on the cursr icon
7. select Open
8. witness nothing occuring.

### Expected Behavior

**Screenshots**

### Your Environment

```
OS: Fedora release 34 (Thirty Four) x86_64 
Kernel: 5.15.11-100.fc34.x86_64 
Resolution: 3440x1440 
DE: GNOME 40.7 
WM: Mutter 
Cursr: https://github.com/bitgapp/Cursr/releases/tag/v1.4.1
```

**Comment by airtonix** [comment]
Oh, had to run `Reset app position` then `open` for it to show up on one of my monitors.

additionally, running it from terminal : 

```
~ 
> ~/Downloads/Cursr.AppImage
load_input_helper [1918]: Using XFree86 keyboard layout.
load_input_helper [1920]: XKB support is required to accurately determine keyboard characters!
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
2022-01-06T22:17:04Z [warn] [0] Invalid EDID format of size 0
2022-01-06T22:17:04Z [warn] [0] Invalid EDID format of size 0
2022-01-06T22:17:04Z [warn] [1] Invalid EDID format of size 0
2022-01-06T22:17:04Z [warn] [1] Invalid EDID format of size 0
2022-01-06T22:17:04Z [warn] [2] Invalid EDID format of size 0
2022-01-06T22:17:04Z [warn] [2] Invalid EDID format of size 0
Failed to retrieve monitor vendor and serial number data for undefined.
Failed to retrieve monitor vendor and serial number data for undefined.
Failed to retrieve monitor vendor and serial number data for undefined.
08:47:04.981 › Checking for update
08:47:06.287 › Update for version 1.4.1 is not available (latest version: 1.4.1, downgrade is disallowed).
2022-01-06T22:17:38Z [warn] [0] Invalid EDID format of size 0
2022-01-06T22:17:38Z [warn] [0] Invalid EDID format of size 0
2022-01-06T22:17:38Z [warn] [1] Invalid EDID format of size 0
2022-01-06T22:17:38Z [warn] [1] Invalid EDID format of size 0
2022-01-06T22:17:38Z [warn] [2] Invalid EDID format of size 0
2022-01-06T22:17:38Z [warn] [2] Invalid EDID format of size 0
Failed to retrieve monitor vendor and serial number data for undefined.
Failed to retrieve monitor vendor and serial number data for undefined.
Failed to retrieve monitor vendor and serial number data for undefined.

```

**Comment by ragauskl** [comment]
@airtonix thank you for the logs, it will be helpful, I'll get to making Cursr compatible with Fedora sometime in first half of this year

I initially made it only for Ubuntu, as it was Linux environment I was most familiar with and from my searches Ubuntu was almost at the top of most popular distributions, also so far there were very few Linux users I did not have time to focus on adding more distributions/testing on more distributions and left it to be dealt with until someone will request support for specific distribution (which you have) - so now it will be in list of to do things

There are few things of high priority things to do for Cursr right now, but once I'm done with them I'll look into making Cursr work on Fedora

**Comment by ragauskl** [comment]
@airtonix v1.4.1 had a bug that is now fixed in v1.4.2 (fix confirmed by other users), application should open as expected
